### PR TITLE
[Merged by Bors] - chore: uncdot up to and including `CategoryTheory`

### PR DESCRIPTION
**Pull request by adomani**
A portion of #12422 containing all the changes in `Mathlib/[A-C]` up to and including `CategoryTheory`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
I already reviewed those in #12422.

File: Mathlib/Analysis/SpecialFunctions/Integrals.lean [hunk 1]
```diff
@@ -381,7 +381,7 @@ theorem integral_rpow {r : ℝ} (h : -1 < r ∨ r ≠ -1 ∧ (0 : ℝ) ∉ [[a,
     ∫ x in a..b, x ^ r = (b ^ (r + 1) - a ^ (r + 1)) / (r + 1) := by
   have h' : -1 < (r : ℂ).re ∨ (r : ℂ) ≠ -1 ∧ (0 : ℝ) ∉ [[a, b]] := by
     cases h
-    · left; rwa [Complex.ofReal_re]
+    · left; rwa [Complex.ofReal_re]https://teams.microsoft.com/v2/
```
Comment (grunweg) [inline Mathlib/Analysis/SpecialFunctions/Integrals.lean:384]:
```suggestion
    · left; rwa [Complex.ofReal_re]

Comment (adomani) [reply Mathlib/Analysis/SpecialFunctions/Integrals.lean:384]: Ooops, how did that happen!?

**Review by joelriou — APPROVED** [review]

**Comment by jcommelin** [comment]
bors d+

bors merge

**Pull request closed without merging**